bucket/lunacy.json

```diff
@@ -19,7 +19,7 @@
     "innosetup": true,
     "checkver": {
         "url": "https://docs.icons8.com/release-notes/",
-        "re": ">([\\d.]+)</h2>"
+        "re": ">(\\d+\\.\\d+\\.\\d+).*<\/h2>"
```

Comment:
```suggestion
        "regex": ">([\\d.]+).*?<\/h2>"
```

Reply:
@Ash258 done

Reply:
Apply everything from details to fix installation